packages/cozy-client/src/registry.js

```diff
@@ -125,6 +125,23 @@ class Registry {
   fetchApp(slug) {
     return this.client.stackClient.fetchJSON('GET', `/registry/${slug}`)
   }
+
+  /**
+   * Fetch the latest version of an app for the given channel and slug
+   *
+   * @param  {object} params - Fetching parameters
+   * @param  {string} params.slug - The slug of the app to fetch
+   * @param  {string} params.channel - The channel of the app to fetch
```

Comment:
/**
 * @typedef {"dev"|"beta"|"stable"} RegistryAppChannel 
 * ...
 * @param {RegistryAppChannel} The channel of the app to fetch
 */

Like that we're better on the type (you can also edit the other occurrence in the file)